pkg/server/web.go

```diff
@@ -18,25 +19,34 @@ type EpEntry struct {
 	Addr  string
 	Iface string
 	IP    string
+	TX_PKT uint64
+	RX_PKT uint64
+	TX_SIZE string
+	RX_SIZE string
 	Login string
 }
 
+
 func listEpEntries(c *gin.Context) {
 	// TODO(shawnlu): Add pkt count
 	var data []*EpEntry
-
 	for user, addr := range users.UserEPMap {
 		c, ok := UPool[addr]
 		if !ok {
 			continue
 		}
-
+		iface_RX_PKT,iface_TX_PKT,iface_RX_SIZE,iface_TX_SIZE,_:=utils.GetRxTxByInterfaceName(c.Iface.Name(),utils.GetLink())
+	
 		data = append(data, &EpEntry{
 			User:  user,
 			Addr:  addr,
 			Iface: c.Iface.Name(),
 			IP:    c.IP,
```

Comment:
Rebase first.
Type of client.IP is net.IP right now, use c.IP.String() to get the values